src/floris/simulation/wake_velocity/gaussianModels/refactor_gauss_mixin.py

```diff
@@ -3,8 +3,8 @@
 import attr
```

Comment:
Is this a modified v2 version of the Gauss wake model?

Reply:
Yup! The `refactor_xx.py` are mostly all waiting to be harmonized with floriz.